core/controllers/editor_test.py

```diff
@@ -208,7 +208,7 @@ def _put_and_expect_400_error(payload):
         # A request with the wrong version number is invalid.
         response_dict = _put_and_expect_400_error(
             _get_payload('New state', version=123))
-        self.assertIn('which is too old', response_dict['error'])
+        self.assertIn('which is not possible', response_dict['error'])
```

Comment:
"not possible" is a bit vague. Explain why it is "not possible".

Reply:
When we read the whole error which is returned then this "not possible" makes sense. Like the whole error will be "Trying to update version 1 of exploration from version 4 which is not possible. Please reload the page and try again.".
So this means that we can not pass the change list from version 4 to update version 1. Does the full statement makes sense or do I still need to make the changes.

Reply:
Ah, I see. Thanks for explaining.